CONTRIBUTING.md

```diff
@@ -0,0 +1,175 @@
+# Contribution Guidelines
+
+This document contains information of use to developers looking to improve this codebase.  See [README.md](https://github.com/tvkitchen/tv-kitchen/blob/master/README.md) for an introduction to this project. All discussions about this project fall under our [code of conduct](CODE_OF_CONDUCT.md).
+
+## Submitting and Reviewing Code
+
+This project has a home on [GitHub](https://github.com/tvkitchen).  Please submit [pull requests](https://help.github.com/articles/about-pull-requests/) (PRs) there.
+
+Please submit changes via pull request, even if you have direct commit access to the repository.  The PR process allows us to get additional eyes on change proposals, and ensures that your changed code [builds cleanly via Travis CI's automated builds](https://travis-ci.org/tvkitchen/tv-kitchen/).
+
+As you work on your branch, try to test it locally to ensure that it still builds and deploys properly, and that you haven't introduced new accessibility bugs. See `docs/TESTING.md` for more instructions.
+
+Generally, the more controversial, complex or large a change, the more opportunity people should have to comment on it.  That means it should garner more comments/approvals, or it means it should sit longer before being merged. You can talk with us about a change you'd like to make before or while you work on it; see [the participation section of the README](README.md#participating). We don't have hard rules about such things, and documentation changes usually don't need to sit as long as functional changes, but figure a business day or two for an average patch to get discussed.
+
+As to when to merge, that's a judgment call.  Usually once an "approved" review goes through, and there aren't any more changes requested, then the author of the PR will merge it (if they have access to push to master).  Generally, wait to merge until the conversation around a change has concluded.  If you're unsure, ask!  "Is this ready to merge?" is often a useful next step in the conversation.
+
+If your PR fixes a bug or adds a feature, please write a test to go with the change (see [TESTING.md](docs/TESTING.md) for details on our testing framework).  If the change involves libraries or would be difficult to test, please use a Given-When-Then description or describe in your PR message how reviewers should test that your change works as expected.
+
+### Document as you go
+
+If you introduce a new framework or dependency, add the necessary information to [INSTALL.md](docs/INSTALL.md) and the other documentation.
+
+### Configurations
+
+We're using `dotenv` for general runtime configuration settings. Never hard code values that should be configurable, but instead edit the .env.template file.
+
+Commits that introduce a new configurable variable should also document the configuration steps in the commit message.
+
+### Migrations
+
+The golden rule of migrations: Never edit a migration file that has been committed to master.
```

Comment:
Is this true globally?

Reply:
I think we could keep this but make it more general -- "For projects with migrations...."